packages/components/src/TimeInput.test.tsx

```diff
@@ -160,12 +173,16 @@ describe('arrow left and right jumps segments', () => {
    * @param {number|Array<number>} movement Keyboard movement to simulate, positive for right, negative for left. Eg. 2 means 2 right arrow presses, -3 means 3 left arrow presses
```

Comment:
You can remove the types from the comments, since the types are specified in the function definition now.